tkg/constants/config_variables.go

```diff
@@ -204,6 +204,8 @@ const (
 	ConfigVariableIPFamily = "TKG_IP_FAMILY"
 	TKGIPV6Primary         = "TKG_IPV6_PRIMARY"
 
+	ConfigVariableNodeIPAMIPPoolName = "NODE_IPAM_IP_POOL_NAME"
```

Comment:
IIUC, this means the user has to apply the pool against the management cluster right? If so, this means that the management cluster is still required to be DHCP, which can trigger the same set of issues we've seen.

Instead can we template out an IPPool CR and expose

```
NODE_IPAM_IP_POOL_SUBNET
NODE_IPAM_IP_POOL_GATEWAY
NODE_IPAM_IP_POOL_START
NODE_IPAM_IP_POOL_END
```

otherwise we'd have to require a pre-created kind cluster/bootstrap cluster

Reply:
Yes, what you said is true. The management cluster is still required to be DHCP. In planning it was decided that we would ship IPAM for workload clusters first. The management cluster IPAM support is intended for a later phase.

We expect there would be considerably fewer management cluster VMs so getting this out first reduces the impact of some of the issues seen while we add support for the management cluster.

Reply:
Yup. Management cluster without DHCP bootstrap will happen in the following release.